internal/service/networkfirewall/firewall_policy.go

```diff
@@ -80,6 +80,20 @@ func ResourceFirewallPolicy() *schema.Resource {
 										Required:     true,
 										ValidateFunc: verify.ValidARN,
 									},
+									"override": {
+										Type:     schema.TypeList,
+										MaxItems: 1,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"action": {
+													Type:         schema.TypeString,
+													Required:     true,
```

Comment:
I realised upon a second look at the API reference that `action` is optional. [Reference](https://docs.aws.amazon.com/network-firewall/latest/APIReference/API_StatefulRuleGroupOverride.html).

```suggestion
													Optional:     true,